src/layer-composer/generators/index.ts

```diff
@@ -11,17 +11,27 @@ import HeatmapGenerator, {
   HEATMAP_COLOR_RAMPS,
 } from './heatmap/heatmap'
 import TrackGenerator, { TRACK_TYPE as TRACK } from './track/track'
+import VesselEventsGenerator, {
+  VESSEL_EVENTS_TYPE as VESSEL_EVENTS,
+  getVesselEventsGeojson,
+} from './vessel-events/vessel-events'
 
-const TYPES = { BASEMAP, CARTO_POLYGONS, BACKGROUND, GL, HEATMAP, TRACK }
+const TYPES = { BASEMAP, CARTO_POLYGONS, BACKGROUND, GL, HEATMAP, TRACK, VESSEL_EVENTS }
 export { TYPES }
 
 export { HEATMAP_GEOM_TYPES, HEATMAP_COLOR_RAMPS }
 
+const dataTransforms = {
```

Comment:
I'm not sure if this will end up being a mess of functions... why not export it directly from the generator ? so each one keeps on their scope.. so instead of 
```import { dataTransforms } from '@globalfishingwatch/layer-composer'```

I suggest having:

```import { getVesselEventsGeojson } from '@globalfishingwatch/layer-composer/generators/vessel-events/'```

Reply:
I agree it's not ideal but I can't get this to work, any idea? 
Using `import { getVesselEventsGeojson } from '@globalfishingwatch/layer-composer/generators/vessel-events/vessel-events'`
```
Module not found: Can't resolve '@globalfishingwatch/layer-composer/generators/vessel-events/vessel-events' in '/Users/erik/Work/gfw/carrier-portal/src/redux-modules/tracks'
```

Reply:
Taking a look on the build folder structure, could you try adding `lib` ?

```import { getVesselEventsGeojson } from '@globalfishingwatch/layer-composer/lib/generators/vessel-events/vessel-events'```

this suggested me also we could have another file to export the utils or data-transform or any name for each generator, like:
```import { getVesselEventsGeojson } from '@globalfishingwatch/layer-composer/lib/generators/vessel-events/our-name'``` to avoid mixing methods with the generator itself